contracts/Staking.sol

```diff
@@ -40,10 +40,12 @@ contract Staking is IStaking, System, IParamSubscriber, IApplication {
   uint256 public constant INIT_RELAYER_FEE = 16 * 1e15;
   uint256 public constant INIT_BSC_RELAYER_FEE = 1 * 1e16;
   uint256 public constant INIT_MIN_DELEGATION = 100 * 1e18;
+  uint256 public constant INIT_TRANSFER_GAS = 2300;
```

Comment:
Please pay attention here if we want to upgrade testnet to the latest code version.

Reply:
thank you